app/src/main/java/uk/gov/android/authentication/integrity/FirebaseAppIntegrityManager.kt

```diff
@@ -62,6 +65,32 @@ class FirebaseClientAttestationManager(
         }
     }
 
+    override fun verifyAttestationJwk(attestation: String): Boolean {
```

Comment:
I  think that it isn't particularly object orientated to be passing things around as string and converting them to objects only when we need to do object comparison, but for the sake of expediency

Reply:
generally agree, but as we discussed, the consumer has no need to do anything with the attestation, so seems overkill to deserialize etc.